lib/vdsm/storage/lvm.py

```diff
@@ -238,7 +238,7 @@ def __getattr__(self, attrName):
  ignore_suspended_devices=1
  write_cache_state=0
  disable_after_error_count=3
- filter=%(filter)s
+ %(filter)s
```

Comment:
This change should be done first, before we add devices support, so when we add devices,
we can switch and use only devices without the filter.